test/bisect/main_test.go

```diff
@@ -210,9 +210,11 @@ type regLayer struct {
 	Size      int64  `json:"size"`
 }
 
-// EscapeImage makes a name safer for filesystem use.
+// escapeImage makes a name safer for filesystem use.
```

Comment:
House style is to use proper capitalization.

Reply:
I figured following the capitalization of the function definition would be preferred. I'll switch it back